src/app/api/v1/share-link/[id]/route.test.ts

```diff
@@ -53,40 +71,101 @@ describe('POST handler', () => {
 
     const request = new Request('http://localhost/api/shlink/123', {
       method: 'POST',
-      body: JSON.stringify({ recipient: 'test@example.com' }),
+      body: JSON.stringify({ recipient: 'test@example.com', managementToken: 'token', passcode: '1234' }),
       headers: { 'Content-Type': 'application/json' },
     });
     const params = { id: '123' };
 
     const response = await POST(request, { params });
```

Comment:
Hi @jacob-khoza-symb, I noticed that the response declaration isn't being used elsewhere in the code. Should we assert the response here to verify its correctness?